content/get-started/transactions.mdx

```diff
@@ -36,61 +32,45 @@ Transaction Fee: 0.00000001
 Find the transaction on https://explorer.ironfish.network/transaction/ee32e78268a1097288503e18510e48d140292ff5ce91d0914ae00f733ad8d166`}
 />
 
-If you want to send a transaction from a different account, you can use the `-f` flag.
-
-E.g.
-
-```sh
-ironfish wallet:send -f MyOtherAccount
-```
-
-### Noninteractive mode
-
-Run `ironfish wallet:send --help` to see the different options available.
-
-E.g. To send 2 coins from the account `Savings`
-
-```sh
-ironfish wallet:send -a 2 -o 0.00000001 -t 997c...7fc52ed -f Savings
-```
+You can also send in non interactive mode by providing flags. Run `ironfish wallet:send --help` to see the different options available. [See more here](/use/get-started/wallet-commands#send-a-transaction).
 
-## Receive a transaction
-To receive a transaction, you just need to communicate the public key of your account to the sender. If you don't know your public key, run the following command:
+## Receive a transactions
+To receive a transaction, give the sender your public key by running the following command:
```

Comment:
nit - this sentence seems to imply that `ironfish wallet:address` will give the sender your public key. Maybe a small change to say that you can view your public key using the command?

Reply:
c